standard/links.lua

```diff
@@ -94,16 +120,22 @@ function Links.transform(links)
         trovo = links.trovo,
         trovo2 = links.trovo2,
         afreeca = links.afreeca,
+        afreeca2 = links.afreeca2,
         weibo = links.weibo,
         vk = links.vk,
         youtube = links.youtube,
         youtube2 = links.youtube2,
+        youtube3 = links.youtube3,
+        youtube4 = links.youtube4,
+        youtube5 = links.youtube5,
         aligulac = links.aligulac,
         rules = links.rules,
         booyah = links.booyah,
         loco = links.loco,
+        dlive = links.dlive,
         tlstream = links.tlstream,
         stream = links.stream,
+        stream2 = links.stream2,
         douyu = links.douyu,
```

Comment:
```suggestion
        douyu = links.douyu or links.douyutv,
```

Reply:
problem is `steam` and `steam2`, they have different links
hence atm it only gets rid of the number for the icon and not for the link retrieval